extensions/amp-date-picker/0.1/amp-date-picker.js

```diff
@@ -481,15 +481,29 @@ export class AmpDatePicker extends AMP.BaseElement {
 
     const min = mutations['min'];
     if (min !== undefined) {
+      this.element.setAttribute('min', min);
       newState['min'] = min;
     }
 
     const max = mutations['max'];
     if (max !== undefined) {
+      this.element.setAttribute('max', max);
       newState['max'] = max;
     }
 
-    this.setState_(newState);
+    let p = null;
+    const src = mutations['src'];
+    if (src !== undefined) {
+      this.element.setAttribute('src', src);
```

Comment:
ditto above, I also verified bind is setting the attribute before calling `mutatedAttributesCallback` so if this is needed, there might be a bug somewhere with bind.

Reply:
I'll double check, thanks for letting me know!